docs/layouts/shortcodes/generated/expert_high_availability_zk_section.html

```diff
@@ -56,11 +56,5 @@
             <td>String</td>
             <td>ZooKeeper root path (ZNode) for job graphs</td>
         </tr>
-        <tr>
```

Comment:
The documentation needs to be regenerated

Reply:
I've already regenerated the doc

Reply:
Yeah, I wasn't aware of the feature where we remove the documentation for deprecated options.